JT/03_java_basics/Java_Programming_basics.adoc

```diff
@@ -0,0 +1,79 @@
+= Introduction to Java
+Liudmila Topal <liudmila.topal@endava.com>
+3.0, July 29, 2022: AsciiDoc article template
+:toc:
+:icons: font
+:url-quickref: https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/
+
+====
+*Java* is a high-level, class-based, object-oriented programming language.
+====
+
+_James Gosling_ at Sun Microsystems (now part of Oracle Corporation) designed it, and it was released in 1995.
+The language was developed with the "*Write Once, Run Anywhere*" (_WORA_) philosophy.
+This principle underscores Java's key feature - *platform independence*, allowing the same Java program to run on multiple platforms without modifications.
+
+*However, to become proficient in any programming language, one Firstly needs to understand the basics of that language.*
+
+Therefore, below are the basics of Java in the format in which it will help you the most to get the heads tart:
+
+Java Environment:::
+The programming *environment* of Java consists of three components mainly:
+.. JDK
+.. JRE
+.. JVM
+
+TIP: More info about Java Environment you can find https://www.geeksforgeeks.org/setting-environment-java/[here].
+
+Comments in Java:::
+In a program, *comments* take part in making the program become more human-readable by placing the detail of code involved
+and proper use of comments makes maintenance easier and finding bugs easily.
+----
+Comments are ignored by the compiler while compiling the code.
+----
+TIP: More info about Comments you can find https://www.geeksforgeeks.org/comments-in-java/[here].
+
+Basic literals:::
+Regardless of its complexity, a program always performs operations on _numbers_, _strings_, and other values.
+These values are called *literals*.
+
+TIP: More info about Literals you can find in *'JT/03_java_basics/basic_literals'* package.
+
+Data Types in Java:::
+Each variable in Java has an associated** data type**.
+Each data type requires different amounts of memory and has some specific operations which can be performed over it.
+
+TIP: More info about Data Types you can find in *'JT/03_java_basics/data_types'* package.
```

Comment:
do you think we can convert this to links?

Reply:
Changed